src/encoded/tests/test_audit_experiment.py

```diff
@@ -2986,14 +2986,72 @@ def test_audit_experiment_tagging_biosample_characterization(
                        {'genetic_modifications': [construct_genetic_modification['@id']]})
     res = testapp.get(base_experiment['@id'] + '@@index-data')
     assert any(error['category'] == 'missing biosample characterization'
-               for error in collect_audit_errors(res, ['ERROR']))                  
+               for error in collect_audit_errors(res, ['ERROR']))
     testapp.patch_json(biosample_characterization['@id'],
                        {'characterizes': biosample_1['@id']})
     res = testapp.get(base_experiment['@id'] + '@@index-data')
     assert all(error['category'] != 'missing biosample characterization'
                for error in collect_audit_errors(res))
 
 
+def test_audit_experiment_pooled_biosample_characterization(
+    testapp,
+    construct_genetic_modification,
+    biosample_characterization,
+    biosample_characterization_no_review,
+    base_experiment,
+    base_target,
+    base_replicate,
+    base_library,
+    base_biosample,
+    biosample_1,
+    biosample_2,
+    award_encode4
+):
+    testapp.patch_json(
+        base_biosample['@id'],
+        {
+            'genetic_modifications': [construct_genetic_modification['@id']],
+            'pooled_from': [biosample_1['@id'], biosample_2['@id']]
+        }
+    )
+    testapp.patch_json(
+        base_library['@id'], {'biosample': base_biosample['@id']}
+    )
+    testapp.patch_json(base_replicate['@id'], {'library': base_library['@id']})
+    testapp.patch_json(
+        base_experiment['@id'],
+        {
+            'assay_term_name': 'ChIP-seq',
+            'award': award_encode4['@id'],
+            'target': base_target['@id']
+        }
+    )
+    res = testapp.get(base_experiment['@id'] + '@@index-data')
+    assert any(
+        error['category'] == 'missing biosample characterization'
+        for error in collect_audit_errors(res, error_types=['ERROR'])
+    )
+    testapp.patch_json(
+        biosample_characterization['@id'],
+        {'characterizes': biosample_1['@id']}
+    )
+    res = testapp.get(base_experiment['@id'] + '@@index-data')
+    assert any(
+        error['category'] == 'missing biosample characterization'
+        for error in collect_audit_errors(res, error_types=['ERROR'])
```

Comment:
I think it would be easier to follow if you make a test fixture of a pooled biosample with characterization or without characterization in parent samples? then the test meaning will be easy to follow.